scripts/run_e2e_tests.py

```diff
@@ -0,0 +1,243 @@
+# Copyright 2019 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS-IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Runs the end to end tests."""
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+from __future__ import unicode_literals  # pylint: disable=import-only-modules
+
+import argparse
+import atexit
+import fileinput
+import os
+import re
+import shutil
+import subprocess
+import sys
+import time
+
+import python_utils
+
+from . import build
+from . import common
+from . import install_chrome_on_travis
+from . import install_third_party_libs
+from . import setup
+from . import setup_gae
+
+_PARSER = argparse.ArgumentParser()
```

Comment:
Pass a `description` argument with the helpful information kept in the `INSTRUCTIONS` sections of the old bash scripts for each migrated script.

```python
argparse.ArgumentParser(description="""
Run this script from the oppia root folder:	

    python -m scripts.run_e2e_tests

The root folder MUST be named 'oppia'.

Note: You can replace 'it' with 'fit' or 'describe' with 'fdescribe' to
run a single test or test suite.
""".strip())
```

Reply:
Done